### DISCUSSION TITLE
All audio tracks at the same time

### DISCUSSION THREAD

**Discussion by fedeiglesias** [iina/iina#3706] [General]
Hi there,
I have a video that have audio tracks for voices and for music in separate tracks, there is a way to reproduce all tracks at the same time ? 

Thanks!

**Comment by low-batt** [comment]
The answer appears to be: Yes

IINA is a graphical user interface on top of the [mpv](https://mpv.io/) media player for the command line. IINA embeds `libmpv` for video and audio support along with libraries from [FFmpeg](https://ffmpeg.org/). Much of the full power of mpv is available in IINA. 

Searching for how to do this with mpv I found mpv issue: https://github.com/mpv-player/mpv/issues/6536

The solution posted uses the [lavfi-complex](https://mpv.io/manual/master/#options-lavfi-complex) filter:
> **--lavfi-complex=\<string>**
>
>    Set a "complex" libavfilter filter, which means a single filter graph can take input from multiple source audio and video tracks.

Audio filters can be set in IINA. See the `Audio` menu. You might have to workaround IINA issues #3620 and #3462.

Another way to set mpv options is through IINA `Preferences`. In the  `Advanced` tab you can `Enable advanced settings` and then add mpv options in the `Additional mpv options` section.
